src/views/add_event.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="">
+  <form action="/insertEvent" method="post">
+      <label for="title-event">Title:</label>
+      <input type="text" name="title" class="pa2 bg-light-gray" placeholder="Name of the event?" id="title-event" required>
+
+      <label for="day-input" class="">Day:</label>
+      <input type="number" name="day" min="1" max="31" value="1" class="bg-light-gray" id="day-input" required>
+
+      <label for="month-input" class="">Month:</label>
+      <input type="number" name="month" min="1" max="12" value="1" class="bg-light-gray" id="month-input" required>
+
+      <label for="year-input" class="">Year:</label>
+      <input type="number" name="year" min="2017" max="2025" value="2017" class="bg-light-gray" id="year-input" required>
+
+      <label for="time-input" class="">Time:</label>
+      <input type="number" name="hour" placeholder="00" min="0" max="23" class="bg-light-gray" id="time-input">:<input type="number" name="minute" placeholder="00" min="0" max="59" class="bg-light-gray" required>
```

Comment:
same comment about the type=number, and I think the placeholder should be hh(hours)and mm(minutes) for more clarification.

Reply:
:+1: